### PR TITLE
feat: Use icon for experiment/trial state [WEB-237]

### PR DESCRIPTION
**Pull request by gt2345**
## Description

Replace state labels in the table with icons. Use animated icons for active states.

These animations should be linked so they all line up if multiple are on the same page at the same time.

Design:
https://www.figma.com/file/g6I9QjuUuoMxe6Gs1oUwuY/State-icons?node-id=1%3A3

Screen shots:
Screen Shot 2022-11-01 at 1 46 10 PM
Screen Shot 2022-11-01 at 1 45 58 PM

## Test Plan

Start a multi trial experiment 
Navigate to experiment list table to view icons, hover to view the actual state
Navigate to trials table to view icons, hover to view the actual state

Edit code to view each icons if needed
Change between dark and light mode to make sure everything looks nice

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by keita-determined** [comment]
just wondering if we need to change filter as well?
image

**Comment by gt2345** [comment]
> just wondering if we need to change filter as well? Ignore this if we dont need to change this
> 
> image

Hi @keita-determined , thank you for bringing it up, I think filters make more sense to keep using text, but it's up to @alexjdetermined  to decide.

**Comment by keita-determined** [comment]
> thank you for bringing it up, I think filters make more sense to keep using text, but it's up to @alexjdetermined to decide.

I just thought it would be nice to have the combination of icon and name like `✅ Complete` instead of the labels.

File: webui/react/src/components/ExperimentIcons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      case RunState.Error:
+      case RunState.Deleted:
+      case RunState.Deleting:
+      case RunState.DeleteFailed:
+        return <img src={error} />;
+      case RunState.Active:
+      case RunState.Unspecified:
+        return <Active />;
+      default:
+        return <img src={isDark ? cancle_dark : cancle_light} />;
```
Comment (julian-determined-ai) [inline webui/react/src/components/ExperimentIcons/index.tsx:48]: **nit:** Update all references of `cancle` to `cancel`

Comment (gt2345) [reply webui/react/src/components/ExperimentIcons/index.tsx:48]: Will do 😅

**Review by julian-determined-ai — APPROVED** [review]
LGTM!

**Pull request merged**